src/appComponents/common/ApiJsonFactory.js

```diff
@@ -1,22 +1,23 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
-
-var commonFactories = angular.module('openevent');
+(function() {
+    "use strict"
+const commonFactories = angular.module('openevent');
 
 commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
-var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
+let baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
     'event/' + config.eventId;
         return {
             getJson: function ($apiEndpoint) {
-                let endpoint;
+                let endpoint ="";
                 if (($apiEndpoint === 'event') && (!config.use_testApi)) {
                     endpoint = baseUrl;
                 } else {
                     endpoint = baseUrl + '/' + $apiEndpoint;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 18 space characters but found 20.](https://www.codacy.com/app/triplez/open-event-webapp/file/2424276792/issues/source?bid=3233857&fileBranchId=3238420#l17)